desktop/backend/python/controller/actions.py

```
@@ -0,0 +1,203 @@
+import shlex
+from typing import List, Tuple
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)